app/src/main/java/com/android/partagix/ui/screens/LoginScreen.kt

```diff
@@ -54,87 +69,197 @@ fun LoginScreen(authentication: Authentication, modifier: Modifier = Modifier) {
   val sheetState = rememberModalBottomSheetState()
   var showSignIn by remember { mutableStateOf(false) }
 
-  Column(
-      modifier = modifier.padding(15.dp).testTag("LoginScreen"),
-      horizontalAlignment = Alignment.CenterHorizontally,
-      verticalArrangement = Arrangement.spacedBy(50.dp, Alignment.CenterVertically),
-  ) {
-    Text(
-        text = "Your neighbors are already here !",
-        modifier = modifier.testTag("LoginTitle"),
-        style =
-            TextStyle(
-                fontSize = 28.sp,
-                fontFamily = FontFamily(Font(R.font.roboto)),
-                fontWeight = FontWeight(500),
-                color = Color(0xFF000000),
-            ))
+  Box(
+      modifier =
+          Modifier.fillMaxSize()
+              .background(MaterialTheme.colorScheme.background)
+              .testTag("Banner")) {
+        Box(
+            modifier =
+                Modifier.fillMaxWidth()
+                    .height(150.dp)
+                    .background(
+                        MaterialTheme.colorScheme.primary,
+                        shape = RoundedCornerShape(bottomStart = 32.dp, bottomEnd = 32.dp))) {
+              Text(
+                  text = "Partagix™",
+                  style = MaterialTheme.typography.displayLarge,
+                  color = MaterialTheme.colorScheme.onPrimary,
+                  modifier = Modifier.align(Alignment.Center))
+            }
+
+        Box(modifier = Modifier.testTag("CenterContent")) {
+          Column(
+              modifier = Modifier.fillMaxSize().padding(16.dp, 160.dp, 16.dp, 16.dp),
+              verticalArrangement = Arrangement.Center,
+              horizontalAlignment = Alignment.CenterHorizontally,
+          ) {
+            Text(
+                text = "Your neighbors are already here !",
+                style =
+                    MaterialTheme.typography.headlineMedium.copy(
+                        fontWeight = FontWeight.SemiBold, textAlign = TextAlign.Center),
+            )
+            val images =
+                listOf(
+                    R.drawable.multiprises,
```

Comment:
Open Question: do we want to have actual images from the database or only "template" images ?
Personally, I think the app would look more active if dynamic images were used.

Reply:
@AntoineBut is working on that and it should be merge soon

Reply:
We can make a future task of replacing these sample images by real ones. But for now, as the images are not working and the db is not filled with real data, I think it's better to do it like this. And as I did it with a parameter image list, it should be easy to replace them.

Reply:
Agreed